### PR TITLE
New documentation page for ui

### PR DESCRIPTION
**Pull request by thfries**
Hi there,

Please find this PR to address #1496
After having some hard time to set up the documentation pages 😉, I started a new page for the ui.

@thjaeckle: Not sure, how to exactly use the docker image. Can you please add a remark where the UI can be reached?

Let me know, if the place in the structure fits as proposed.
This is just a first draft, that needs to be refined I think.

Kind regards, Thomas

### CODE REVIEW

File: documentation/src/main/resources/pages/ditto/user-interface.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+{% include image.html file="pages/ui/screenshot_introduction.png" alt="UI example screenshot" caption="Ditto™ Explorer UI" max-width=800 url="https://eclipse.github.io/ditto/index.html?primaryEnvironmentname=ditto_sandbox" %}
+
+
+## How to use
+
+The latest version of the Explorer UI is available online [here](https://eclipse.github.io/ditto/index.html?primaryEnvironmentname=ditto_sandbox)
+
+There is also a Docker image prepared to server the latest version by docker:
+```bash
+docker pull eclipse/ditto-ui:nightly
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/user-interface.md:24]:
```suggestion
# to start the latest released UI version:
docker run -p 8088:8080 eclipse/ditto-ui

# or to start the latest nightly instead:
docker run -p 8088:8080 eclipse/ditto-ui:nightly
```

Afterwards, access the Ditto UI in your browser at `http://localhost:8088`

File: documentation/src/main/resources/pages/ditto/user-interface.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+You can use the Explorer UI with different Ditto™ instances and switch easily between them.
+
+For example you can work with your **local running Ditto™ instance** and the remote Eclipse Ditto™ **sandbox instance**.
+Another example would be to use different stages of Ditto™ like **development**, **test** and **production**.
+
+### Create a new environment
+To create a new environment you just need to specify
+* a unique name
+* the api URI to reach Ditto™
+* If it is a Ditto 2 or ditto 3 Version (only relevant if you want to use the connectivity API of Ditto™ prior to version 3)
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/user-interface.md:37]:
```suggestion
* whether it is a Ditto 2 or Ditto 3 version (only relevant if you want to use the connectivity API of Ditto™ prior to version 3)
```

File: documentation/src/main/resources/pages/ditto/user-interface.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Selecting an environment
+Select an environment in the drop down in the upper right:
+{% include image.html file="pages/ui/screenshot_select_environment.png" alt="Screenshot how to select an environment" caption="Select a new environment" max-width=400 %}
+
+
+### Control environments by URL parameters
+The following query parameters can be used to control environments
+
+| URL search parameter | Description |
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/user-interface.md:49]:
```suggestion
| URL query parameter | Description |
```

**Review by thjaeckle — CHANGES_REQUESTED** [review]
Thanks @thfries for providing the documentation.

I added some inline feedback .. and added how to start the Docker container.
There I noticed that for the Ditto 3.0.0 release the "latest" tag was not uploaded to Docker-Hub and I fixed that :)

**Comment by thfries** [comment]
Hi @thjaeckle,
thanks for the feedback. I processed all remarks. Links with the pre-selected sandbox also work 😀

Open points from my view:
* Shall we review other pages if referring to the ui would make sense?
* Some general quick-start guide on the UI?

But other than that I m ok to start with this page as is. wdyt?

**Review by thjaeckle — APPROVED** [review]
> Hi @thjaeckle, thanks for the feedback. I processed all remarks. Links with the pre-selected sandbox also work 😀
> 
> Open points from my view:
> 
> * Shall we review other pages if referring to the ui would make sense?
> * Some general quick-start guide on the UI?
> 
> But other than that I m ok to start with this page as is. wdyt?

I don't think that adding cross-references to the UI documentation page from many other pages does bring much benefit.
I guess someone searching for a UI will find the new documentation page.

A Quick-Start guide would be good, yes. 
But I would also see that as an add-on to the added documentation.

FMPOV good to merge 👍 
Thanks 🎉

**Pull request merged**